go/ql/src/experimental/CWE-770/DenialOfService.ql

```diff
@@ -0,0 +1,64 @@
+/**
+ * @name Denial Of Service
+ * @description slices created with the built-in make function from user-controlled sources using a
+ * maliciously large value possibly leading to a denial of service.
+ * @kind path-problem
+ * @problem.severity high
+ * @security-severity 9
+ * @id go/denial-of-service
+ * @tags security
+ *       experimental
+ *       external/cwe/cwe-770
+ */
+
+import go
+
+class BuiltInMake extends DataFlow::Node {
```

Comment:
`BuiltInMake` makes me think this is just `Builtin::make()`. How about `LengthArgumentToMake`? Also this class needs a QLDoc. Or, as it's only used in one place, you could just inline it.

Reply:
I'll inline it.